### DISCUSSION TITLE
PCB for AcidBox

### DISCUSSION THREAD

**Discussion by streetuff** [copych/AcidBox#9] [Show and tell]
Hey!

First time on Github here. Over the past weeks i created a PCB with easyeda for AcidBox. Works perfectly with hairless midi or "real" midi :-D

The Board supports the Lolin ESP32S3 (not the pro version), the PCM5102 DAC, Audio out via 3.5mm jack and cinch/rca. Midi In/out/thru via DIN5.

You can find the gerbers, bom and some pictures here -> https://github.com/streetuff/AcidBox-PCB

**Comment by copych** [comment]
@streetuff wow! Great work! I will put a link to your repo if you don't mind.

**Reply by streetuff** [reply]
Sure. Why not? Just not used to Github and how to "handle" stuff here yet. But i hope this helps others building up on your project.

**Reply by streetuff** [reply]
Tomorrow i will try a pull request. got the RGB-LED on the lolin Blinking on every 1st/4th beat in a random color. Looks kinda cool.

**Comment by Maitresinh** [comment]
So great  of you.

Just a question : is that ESP32 would work as a LOLIN S 3 ?

https://fr.aliexpress.com/item/1005006211815608.html?spm=a2g0o.detail.0.0.7093w9FNw9FNoJ&gps-id=pcDetailTopMoreOtherSeller&scm=1007.40050.362094.0&scm_id=1007.40050.362094.0&scm-url=1007.40050.362094.0&pvid=06b4b43d-be8f-4559-9238-1e2d9fcdfcce&_t=gps-id:pcDetailTopMoreOtherSeller,scm-url:1007.40050.362094.0,pvid:06b4b43d-be8f-4559-9238-1e2d9fcdfcce,tpp_buckets:668%232846%238108%231977&pdp_npi=4%40dis%21EUR%2111.77%213.76%21%21%2189.60%21%21%40211b5db217008132204631646e4b78%2112000036338360731%21rec%21FR%212109886745%21

**Reply by streetuff** [reply]
Hi! Just checked the Pinout of this Board. It is not pin compatible with the Lolin S3.
Also the Lolin S3 Boards usually cost ~ 15 - 20€. Here is the Seller i usually buy -> https://www.ebay.de/itm/295854503703

**Reply by Maitresinh** [reply]
Noted, thanks.
What about the RCAs ?  I don't want to mess with a bad formfactor, and could find them on google

**Reply by streetuff** [reply]
I dont have a partnumber for the RCAs. I used, what i had still laying around here from another project.
But you really dont need the RCAs, because you got a 3.5mm audio out jack on the PCM5102 DAC module.

**Reply by Maitresinh** [reply]
I saw it.
So all good

**Reply by Maitresinh** [reply]
Anyway, found

https://fr.aliexpress.com/item/1005003701964597.html?src=google&src=google&albch=shopping&acnt=248-630-5778&slnk=&plac=&mtctp=&albbt=Google_7_shopping&gclsrc=aw.ds&albagn=888888&isSmbAutoCall=false&needSmbHouyi=false&src=google&albch=shopping&acnt=248-630-5778&slnk=&plac=&mtctp=&albbt=Google_7_shopping&gclsrc=aw.ds&albagn=888888&ds_e_adid=&ds_e_matchtype=&ds_e_device=c&ds_e_network=x&ds_e_product_group_id=&ds_e_product_id=fr1005003701964597&ds_e_product_merchant_id=109217176&ds_e_product_country=FR&ds_e_product_language=fr&ds_e_product_channel=online&ds_e_product_store_id=&ds_url_v=2&albcp=20180143323&albag=&isSmbAutoCall=false&needSmbHouyi=false&gad_source=1&gclid=CjwKCAiAsIGrBhAAEiwAEzMlC-OWr-u97VyLW-cOK-a-wuEikItTt7xatyOHBToTEQake8gDgqvdRBoCVAcQAvD_BwE&aff_fcid=4c485661bd894b3090911359164ed174-1700838349317-08963-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=4c485661bd894b3090911359164ed174-1700838349317-08963-UneMJZVf&terminal_id=8f5d7a0ebfda47398e42d03461d1f45b&afSmartRedirect=y
